src/transformers/trainer.py

```diff
@@ -2177,11 +2177,20 @@ def _load_best_model(self):
         logger.info(f"Loading best model from {self.state.best_model_checkpoint} (score: {self.state.best_metric}).")
         best_model_path = os.path.join(self.state.best_model_checkpoint, WEIGHTS_NAME)
         best_safe_model_path = os.path.join(self.state.best_model_checkpoint, SAFE_WEIGHTS_NAME)
+        best_adapter_model_path = os.path.join(self.state.best_model_checkpoint, "adapter_model.bin")
+        best_safe_adapter_model_path = os.path.join(self.state.best_model_checkpoint, "adapter_model.safetensors")
```

Comment:
Those two should be in constants (like `WEIGHTS_NAME`) as they are now used several time across the file.

Reply:
makes sense, just added it!